### PR TITLE
two lines integration + flowplayer

### LINKED ISSUE #42: fix videojs5 plugin script name
the correct script name is `peer5.videojs5.plugin.js`
https://docs.peer5.com/players/videojs/

### LINKED ISSUE #44: Update players integration guides
add flowplayer as well

### PR DESCRIPTION
**Pull request by mrbar42**
fixes #42
fixes #44

also improved nginx guide binary reference path sue to feedback from customers

### CODE REVIEW

File: docs/players/videojs.md [hunk 1]
```diff
@@ -10,9 +10,10 @@ The integration with Peer5 plugin is as easy as it can get.
 In addition to the player script, include Peer5 client and VideoJS plugin.
  
 ## Peer5 client and plugins scripts
+add these two scripts to the `head` of your player's page
 
      <script src="//api.peer5.com/peer5.js?id=PEER5_API_KEY"></script>
-     <script src="//api.peer5.com/peer5.videojs5.hlsjs.plugin.js">
+     <script src="//api.peer5.com/peer5.videojs5.plugin.js">
```
Comment (shacharz) [inline docs/players/videojs.md]: missing </script>

File: docs/players/videojs.md [hunk 2]
```diff
@@ -10,9 +10,10 @@ The integration with Peer5 plugin is as easy as it can get.
 In addition to the player script, include Peer5 client and VideoJS plugin.
  
 ## Peer5 client and plugins scripts
+add these two scripts to the `head` of your player's page
```
Comment (shacharz) [inline docs/players/videojs.md]: There's some kind of bug with this videojs script:
http://codepen.io/anon/pen/EgWOQm

Comment (mrbar42) [inline docs/players/videojs.md]: checkout Peer5/peer5-ws#1400

File: docs/players/flowplayer.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ The following information needs to be filled according to your actual data:
 
         <!-- Flowplayer scripts & styles -->
         <script src="//releases.flowplayer.org/6.0.5/flowplayer.min.js"></script>
-        <link href="//releases.flowplayer.org/6.0.5/skin/minimalist.css" rel="stylesheet">
+        <link href="//releases.flowplayer.org/6.0.5/skin/minimalist.css" rel="stylesheet" />
```
Comment (shacharz) [inline docs/players/flowplayer.md]: I don't think link tag needs to end with /> just >

Comment (mrbar42) [inline docs/players/flowplayer.md]: fixed

**Pull request merged**